types/onfido-sdk-ui/onfido-sdk-ui-tests.ts

```diff
@@ -0,0 +1,52 @@
+import { init, iOnfidoResponse } from 'onfido-sdk-ui';
+
+const fidoObj = init({
+    token: 'someTokenHere',
+    containerId: 'onfido-mount',
+    onComplete: (data: iOnfidoResponse) => {
+        // Some completion here..
```

Comment:
Would it be interesting to access properties of `data`?

Reply:
I don't think so in this case, normally you just read out the result for use inside the app.